doc/urgency.md

```diff
@@ -0,0 +1,12 @@
+# Urgency
+
+Urgency is pretty primitive at the moment. It should only affect tasks that are
+not resolved.
+
+It takes the Task's priority as a Fibonacci (Low = 1, Critical = 5) and an applies a multiplier (5).
+
+If the status is active, it adds 5.
+
+If a project or tag is applied, add 3 for each.
+
+It also multiplies the age of the task by 0.05 and adds that to the urgency.
```

Comment:
should it not take into account the due date and the task priority?

Reply:
Priority is taken into account.  After adding urgency, it would sort by priority > urgency > created.

I didn't do anything with due date because I understood it wasn't implemented, but it shouldn't be too hard to add that.